scripts/vim-patch.sh

```diff
@@ -191,10 +195,14 @@ preprocess_patch() {
   # Remove "Last change ..." changes in doc files.
   2>/dev/null $nvim --cmd 'set dir=/tmp' +'%s/^@@.*\n.*For Vim version.*Last change.*\n.*For Vim version.*Last change.*//' +w +q "$file"
 
-  # Remove screen dumps, testdir/Make_*.mak files
-  local na_src_testdir='Make_amiga.mak\|Make_dos.mak\|Make_ming.mak\|Make_vms.mms\|dumps/.*.dump'
+  # Remove gui, option, setup, screen dumps, testdir/Make_*.mak files
+  local na_src_testdir='gen_opt_test.vim\|gui_.*\|Make_amiga.mak\|Make_dos.mak\|Make_ming.mak\|Make_vms.mms\|dumps/.*.dump\|setup_gui.vim'
   2>/dev/null $nvim --cmd 'set dir=/tmp' +'g@^diff --git a/src/testdir/\<\%('"${na_src_testdir}"'\)\>@norm! d/\v(^diff)|%$
' +w +q "$file"
 
+  # Remove testdir/test_*.vim files
+  local na_src_testdir='balloon.*\|channel.*\|crypt.vim\|gui.*\|job_fails.vim\|json.vim\|mzscheme.vim\|netbeans.*\|popupwin.*\|prompt_buffer.vim\|restricted.vim\|shortpathname.vim\|tcl.vim\|terminal.*\|xxd.vim'
```

Comment:
Why prompt_buffer.vim?  I thought that functionality got ported.

Reply:
There's a feature check at the top of the file. Some tests fail after commenting out the feature checks. Many tests depend on Vim's terminal.

Reply:
On second thought, I'm removing test_prompt_buffer.vim here. It could be ported later to run some of the tests.